maps/wasteland/market.lua

```diff
@@ -210,11 +214,20 @@ local function set_offers(town_center)
     for _, item in pairs(market_items) do
         market.add_market_item(item)
     end
+    town_center.offers = market.get_market_items() -- I'm lazy to refactor everything
```

Comment:
This comment will be old quickly

Reply:
There's no reason to have market.get_market_items, but it requires more refactoring.

Reply:
Pls remove the comment or change to "This can be optimised out"

Reply:
I changed the comment to "get rid of it by refactoring data" 5586e87